cargo-dist/src/cargo_build.rs

```diff
@@ -62,32 +62,17 @@ impl<'a> DistGraphBuilder<'a> {
                 rustflags.push_str(" -Ctarget-feature=+crt-static -Clink-self-contained=yes");
             }
 
-            // If we're trying to cross-compile on macOS, ensure the rustup toolchain
+            // If we're trying to cross-compile, ensure the rustup toolchain
             // is setup!
-            if target.ends_with("apple-darwin")
-                && self.inner.tools.cargo.host_target.ends_with("apple-darwin")
-                && target != self.inner.tools.cargo.host_target
+            if target != self.inner.tools.cargo.host_target
             {
```

Comment:
Mind moving this onto the previous line? `rustfmt` flags this

Reply:
Opps, I did not notice that CI failure. I ran `cargo fmt` and pushed.

Reply:
Thanks!